src/dodal/devices/attenuator/attenuator.py

```diff
@@ -0,0 +1,136 @@
+from typing import Optional
+
+from ophyd import Component, Device, EpicsSignal, EpicsSignalRO, Signal
+from ophyd.status import Status, SubscriptionStatus
+
+from dodal.devices.attenuator.filter import AtteunatorFilter
+from dodal.devices.detector import DetectorParams
+from dodal.devices.status import await_value
+from dodal.log import LOGGER
+
+
+class Attenuator(Device):
+    class TransmissionSignal(Signal):
+        def set(self, value, *, timeout=None, settle_time=None, **kwargs):
+            return self.parent.set_transmission()
+
+    do_set_transmission: TransmissionSignal = Component(TransmissionSignal)
```

Comment:
Should: I think we can just call this transmission

Reply:
@neil-i03 how often do you want to touch any of the underlying PVs on the attenuator? I assume 99% of the time you're just setting transmission?

Reply:
I also think it needs a comment that it is in the range 0-1, people may expect it to be a percentage

Reply:
As discussed, I think we can move the set override to the Attenuator itself so we can do something like `attenuator.set(desired_transmission)`

Reply:
This is changed now